internal/app/machined/pkg/runtime/sequencer.go

```diff
@@ -34,31 +34,36 @@ const (
 	SequenceRecover
 	// SequenceNoop is the noop sequence.
 	SequenceNoop
+	// SequenceApplyConfiguration is the apply configuration sequence.
+	SequenceApplyConfiguration
```

Comment:
nit: should we sort the constants alphabetically as well?

I think those constants never get out of the running system, so it should be safe to change them

Reply:
Yeah, I didn't want to reorder the enum, but you are correct; I don't think these are exposed outside of a release.  It's still not something I was comfortable changing.